app/View/Events/add.ctp

```diff
@@ -11,7 +11,8 @@
                     'field' => 'org_id',
```

Comment:
Not sure about the purpose of this field. Maybe @iglocska could shed some light?

Reply:
removed, as discussed with @mokaddem and @iglocska the field is not required.